prod-stack.py

```diff
@@ -770,6 +775,23 @@
             },
         ]
     },
+    {
+        'name': 'Mirrorer',
+        'command': 'mirrorer',
+        'memory': '156',
```

Comment:
We can drop the memory and let the defaults handle it (we're getting tight on allocated memory for the service)